design/design.md

```diff
@@ -0,0 +1,112 @@
+Applications?
+mithril, peras, leios
+
+What cryptographic primitives will we implement?
+I propose that we implement the generic ALBA primitive to make it as applicable
+as possible.
+This means there is only a generic notion of "elements" and no notions of
+signatures, vrf, etc.
+This would be somewhat difficult to work with for all our three applications
+that really just need a weighted multisignature scheme.
+Hence, my second proposal is to use this generic ALBA implementation to implement
```

Comment:
Raphael: put the multisignature scheme in a different repository.

Reply:
I support of having a directory `examples/` to show specific examples of usage of Alba in different use cases (that may go beyond multisignature schemes).